.github/release-please-config.md

```diff
@@ -203,27 +248,69 @@ Release-please will create ONE PR that:
 
 When merged, three separate GitHub releases will be created automatically.
 
+### Beta Release Example
+
+If you make these commits:
+
+```bash
+feat(next): add Input component
+fix(next): fix Placeholder styling
+```
+
+Release-please will include beta changes in the release PR:
+
+- Title includes "next": `chore: release eds-core-react-next 2.0.1-beta.1`
+- Bumps beta version: `2.0.1-beta.0` → `2.0.1-beta.1`
+- Updates `src/components/next/CHANGELOG.md`
+- When merged, publishes to `@equinor/eds-core-react@beta`
+
+**Note:** With `"separate-pull-requests": false`, both stable and beta changes can appear in the **same PR** if you have commits for both. The PR will contain updates to both `CHANGELOG.md` and `src/components/next/CHANGELOG.md`.
```

Comment:
Where is this separate-pull-request configured @pomfrida ?

Reply:
It is configured in the release-please-config.json (not in this PR but before)